pypeman/contrib/http.py

```diff
@@ -302,7 +302,11 @@ async def handle_request(self, msg):
         data = None
         if method.lower() in ['put', 'post', 'patch']:
             data = msg.payload
-        async with aiohttp.ClientSession(connector=conn, cookies=cookies) as session:
+        async with aiohttp.ClientSession(
+            connector=conn,
+            cookies=cookies,
+            trust_env=True,  # respect e.g. http_proxy env vars
```

Comment:
After the discussion yesterday I am wondering whether we should add the version of aiohttp in the requirements. (trust_env was introduced in 2.3. So no issues for this particular change)

Reply:
might add >=2.3 to requirements just to be sure.

Reply:
2.3 dates from 2017. so better to keep requirements simple instead of adding this constraint

Reply:
> 2.3 dates from 2017. so better to keep requirements simple instead of adding this constraint

Makes sense, thanks.